lib/editor/util/__tests__/__snapshots__/map.js.snap

```diff
@@ -1030,76 +797,32 @@ Object {
   "coordinates": Array [
     Array [
       Array [
-        -77.922203,
-        42.086407,
+        -77.9230535030365,
+        42.096176462786296,
```

Comment:
The same odd thing in `should update the first stop` is now happening in reverse in the `should update the last stop`.

Before the test:

<img width="224" alt="Screen Shot 2019-04-03 at 3 11 40 PM" src="https://user-images.githubusercontent.com/3112493/55516800-36846080-5623-11e9-9c96-c879dcb642a4.png">

After:

<img width="239" alt="Screen Shot 2019-04-03 at 3 11 23 PM" src="https://user-images.githubusercontent.com/3112493/55516801-36846080-5623-11e9-9588-cdef1c534e60.png">

Zoomed in the first point is now going backwards to some other point and going forward.

<img width="556" alt="Screen Shot 2019-04-03 at 3 13 19 PM" src="https://user-images.githubusercontent.com/3112493/55516792-31271600-5623-11e9-8fde-e9f495f5d84f.png">